components/cursor/cursor.app.mjs

```
@@ -1,11 +1,145 @@
+        prevContext, status,
+      }) {
+        const {
+          agents, nextCursor,
+        } = await this.listAgents({
+          params: {
+            cursor: prevContext?.cursor,
+          },
+        });
+        let options = [];
+        if (status) {
+          options = agents?.filter(({ status }) => status === status);
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)